recipe/meta.yaml

```diff
@@ -245,11 +250,14 @@ outputs:
       run:
         - _libgcc_mutex {{ _libgcc_mutex }}
         - {{ pin_subpackage("libgomp", max_pin=None) }}
+      run_constrained:
+        # conflict with previous name
+        - openmp_impl 9999
     build:
       string: 0_gnu
       run_exports:
         strong:
-          - {{ pin_subpackage("openmp_impl", max_pin=None) }}
+          - {{ pin_subpackage("_openmp_mutex", max_pin=None) }}
```

Comment:
I am not sure we want/need this run export. People are supposed to depend on libgomp in the host section to insert the openmp_mutex run export. I don't think we need to make this mutex itself self-propagating.

Reply:
When conda-build gives a warning about `_openmp_mutex` being not in run, people might just use this instead of libgomp. Anyways, there's no harm having this here.

Reply:
We had the run export for the llvm mutex on itself ignored, so we should make sure to put it back.